src/main/webapp/app/modules/conflicts/shared/components/service/services-details.tsx

```diff
@@ -29,6 +30,16 @@ export class ServicesDetails extends React.Component<IServicesDetailsProp, IServ
     this.setState({ serviceNumber });
   };
 
+  getServiceDisplayLabel = (s, index) => {
+    if (!!s.service && !_.isEmpty(s.service.name)) {
+      return { value: index, label: s.service.name };
+    }
+    return {
+      value: index,
+      label: `${translate('serviceNetApp.service.detail.title')} ${index}`
```

Comment:
Using the index here won't give us certainty that this number will be increased by 1 for each case (we might have labels which could mislead users). I suggest that this value should be incremented by 1 always.